src/blocks/HeaderBlockBrandPivot/index.tsx

```diff
@@ -0,0 +1,366 @@
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+import { ContextContainer } from 'components/containers/ContextContainer'
+import * as React from 'react'
+import { AppLink } from '../../components/AppLink'
+import { ContentWrapper } from '../../components/blockHelpers'
+import {
+  ButtonLinkBrandPivot,
+  ButtonStyleType,
+} from '../../components/ButtonBrandPivot/Button'
+import { Togglable } from '../../components/containers/Togglable'
+import { HedvigLogotype } from '../../components/icons/HedvigLogotype'
+import {
+  GlobalStory,
+  GlobalStoryContainer,
+  LinkComponent,
+} from '../../storyblok/StoryContainer'
+import { getStoryblokLinkUrl } from '../../utils/storyblok'
+import {
+  BaseBlockProps,
+  MinimalColorComponent,
+  minimalColorComponentColors,
+} from '../BaseBlockProps'
+import { MenuItem } from './MenuItem'
+import { Burger, TABLET_BP_DOWN, TABLET_BP_UP } from './mobile'
+
+export const WRAPPER_HEIGHT = '6rem'
+export const HEADER_VERTICAL_PADDING = '1.2rem'
+export const TOGGLE_TRANSITION_TIME = 250
+
+const isBelowScrollThreshold = () =>
+  typeof window !== 'undefined' && window.scrollY > 20
+
+const Wrapper = styled('div')<{ inverse: boolean; open: boolean }>(
+  ({ inverse, open }) => ({
+    position: 'fixed',
+    top: 0,
+    left: 0,
+    right: 0,
+    zIndex: 100,
+    color: inverse ? colorsV3.white : colorsV3.gray900,
+    transition: 'color 300ms',
+
+    [TABLET_BP_DOWN]: {
+      bottom: open ? 0 : undefined,
+    },
+  }),
+)
+const Filler = styled('div')({
+  height: WRAPPER_HEIGHT,
+})
+const HeaderBackgroundFiller = styled('div')<{ transparent: boolean }>(
+  ({ transparent }) => ({
+    position: 'absolute',
+    top: 0,
+    left: 0,
+    right: 0,
+    zIndex: -1,
+    height: WRAPPER_HEIGHT,
+    backgroundColor: colorsV3.white,
+    opacity: transparent ? 0 : 1,
+    transition: 'opacity 300ms',
+  }),
+)
+
+const InnerHeaderWrapper = styled('div')({
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  width: '100%',
+  height: WRAPPER_HEIGHT,
+  padding: HEADER_VERTICAL_PADDING + ' 0',
+})
+const Menu = styled('ul')<{ open: boolean }>(({ open }) => ({
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  margin: 0,
+  padding: 0,
+  listStyle: 'none',
+
+  [TABLET_BP_DOWN]: {
+    position: 'absolute',
+    display: 'block',
+    zIndex: 101,
+    width: '80%',
+    height: '100vh',
+    top: 0,
+    bottom: 0,
+    right: open ? '20%' : '100%',
+    paddingTop: `calc(${WRAPPER_HEIGHT} + ${HEADER_VERTICAL_PADDING})`,
+    fontSize: 18,
+    background: colorsV3.white,
+    transition: `right ${TOGGLE_TRANSITION_TIME}ms`,
+    color: colorsV3.gray900,
+    overflow: 'scroll',
+    WebkitOverflowScrolling: 'touch',
+  },
+}))
+
+const LogoLink = styled('a')({
+  display: 'inline-flex',
+  paddingTop: 3, // fix to push down logo a little or it looks unbalanced
```

Comment:
do we need this with the new logo?

Reply:
Ah good catch! 🙌 It actually skews that logo since the height is set to auto so let's remove it. Will  adjust it later if we see that needs to be re-aligned in mobile views